### DISCUSSION TITLE
Instillation FoooXus-Fooocus-Extender

### DISCUSSION THREAD

**Discussion by bsdesigner** [toutjavascript/FoooXus-Fooocus-Extender#3] [General]
.

Sorry, could you please record a video tutorial on how to install the FoooXus-Fooocus-Extender, because there is no way to install....

**Comment by toutjavascript** [comment]
I will do in few days
Can you tell me what errors you got ?

**Reply by bsdesigner** [reply]
Like you, I use Fooocus in Stability Matrix, but I don't know Python at all and have no idea what folders to put where, what to prescribe and what to activate. Apparently, I'm a complete ignoramus in this... ((

**Reply by toutjavascript** [reply]
Ok, I understand.
It could be very tedious to install python, git, pip and so on

What do you think of a .exe file to download without no need to install anything else ?

**Reply by bsdesigner** [reply]
That would be great!!!

**Reply by toutjavascript** [reply]
I am working on it.
I manage to build a standalone .exe file that contains all the modules.

I will publish the release tomorrow if i do not encounter new issue

**Comment by dog0311usmc** [comment]
I know I saw the 7zip password somewhere but now I can't find it. I've been up all night adding stuff. Any chance you can ell me where I saw the password on the full exe file?
